engines/stark/ui/world/gamescreen.cpp

```diff
@@ -65,14 +65,15 @@ GameScreen::~GameScreen() {
 }
 
 void GameScreen::open() {
-	g_engine->_mixer->pauseAll(false);
+	if (g_engine->isPaused()) {
+		g_engine->pauseEngine(false);
+	}
 	StarkUserInterface->freeGameScreenThumbnail();
 }
 
 void GameScreen::close() {
 	_cursor->setMouseHint("");
-	g_engine->_mixer->pauseAll(true);
-	StarkUserInterface->saveGameScreenThumbnail();
+	g_engine->pauseEngine(true);
 }
```

Comment:
It seems that the original `pauseEngine()` is well enough. No need to called `saveGameScreenThumbnail()` now since it will be called in the `pauseEngine()`.